src/components/response/ResponseSwitcher.tsx

```diff
@@ -231,7 +235,7 @@ export function ResponseSwitcher({
         />
       )}
 
-      {(response.withDivider || configInUse.responseDividers) && <Divider mt="xl" mb="xs" />}
+      {(response.withDivider || responseDividers) && <Divider mt="xl" mb="xs" />}
```

Comment:
Seems you already deal with this above on 118.
```suggestion
      {responseDividers && <Divider mt="xl" mb="xs" />}
```

Reply:
Updated